### ISSUE TITLE
Clock only

### ISSUE THREAD

**Issue by Mefik420** [mtricht/gamesense-essentials#69]
I would like to have only clock visible for the whole time.
Could you add enable/disable viewing actual listened song option? Or which files/lines in files I need to remove to have only clock available. Thank you for your work ❤️

**Comment by mtricht** [comment]
@Mefik420 Good idea, added in 1.11.0: https://github.com/mtricht/gamesense-essentials/releases/tag/1.11.0
